app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,54 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserLoginData;
+import com.codesoom.assignment.errors.LoginNotMatchPasswordException;
+import com.codesoom.assignment.errors.UserNotFoundException;
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.springframework.stereotype.Service;
+
+/**
+ * 인증, 인가와 관련된 처리를 담당합니다.
+ */
+@Service
+public class AuthenticationService {
+
+    private final JwtUtil jwtUtil;
+    private final UserRepository userRepository;
+
+    public AuthenticationService(JwtUtil jwtUtil, UserRepository userRepository) {
+        this.jwtUtil = jwtUtil;
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * User 로그인 정보를 확인해 access token 발급합니다.
+     *
+     * @param userLoginData email과 password를 담은 User 정보
```

Comment:
`userLoginData`에 OTP 정보 같은 것이 추가된다면 이 주석도 수정이 되어야겠죠. 이렇게 하는 정도가 적절해 보입니다.
```suggestion
     * @param userLoginData 로그인에 필요한 user 정보
```

Reply:
너무 구체적인 내용이 적혀 있네요.